.github/workflows/windows-build.yml

```diff
@@ -32,12 +33,40 @@ jobs:
         if: ${{ matrix.compiler == 'msvc' }}
         run: |
           copy env.vs2022.bat env.bat
+          $file = ".\env.bat"
```

Comment:
These stuff is too hacky.

Reply:
Use ci creat env.bat and use
https://github.com/rime/librime/pull/718/commits/bb616d946e8516b7bf735c9f44ff14ea306ca14b